client/src/totalRatings.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+
+const TotalRatings = ({ total, average }) => {
+  const stars = [];
+  for (let index = 1; index <= 5; index += 1) {
+    const color = index <= average ? 'gold' : 'grey';
+    stars[index] = <FontAwesomeIcon key={`star-${index}`} icon={faStar} color={color} />;
```

Comment:
Pretty neat that you directly added the review star icon to jsx file!

Reply:
Thanks :)